packages/react/src/date-picker/DateRangePicker.test.tsx

```diff
@@ -455,8 +457,8 @@ describe('DateRangePicker', () => {
 		await userEvent.type(await getFromInput(), fromValidDateAsString);
 		await expect(await getFromInput()).toHaveValue(fromValidDateAsString);
 
-		await userEvent.type(await getFromInput(), toValidDateAsString);
-		await expect(await getFromInput()).toHaveValue(toValidDateAsString);
+		await userEvent.type(await getToInput(), toValidDateAsString);
```

Comment:
Oops this was my mistake, we should be typing in the `to` date field here, not the `from` field.